<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ButtonMakeRandomBoard">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>490</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>make random board</string>
    </property>
   </widget>
   <widget class="QWidget" name="boardWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="playerTurn">
     <item>
      <widget class="QLabel" name="PlayersTurn">
       <property name="text">
        <string>White to move</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>569</x>
      <y>69</y>
      <width>121</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lableEatenPiecesBlack">
       <property name="text">
        <string>Captured pieces black</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lableEatenPiecesWhite">
       <property name="text">
        <string>Captured pieces white</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layoutCheckMate">
     <item>
      <widget class="QLabel" name="lableShowCheckMate">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
